### ISSUE TITLE
Running my own instance. 

### ISSUE THREAD

**Issue by Devl326789** [Athou/commafeed#956]
I'm trying to run my own instance. I have used the public version for a little while and while it works great it appears to be very overloaded and slow. Regardless, thanks for hosting it to trial it out. 

I have followed the steps here: https://david.sferruzza.fr/blog/2014-08-18-how-to-install-commafeed-rss-reader/

I have installed and tested MySQL which appears to work fine, except I cannot get around this error `[2021-01-30 18:59:18,301] org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator: HHH000342: Could not obtain connection to query metadata : Could not create connection to database server. Attempted reconnect 20 times. Giving up.`

I've also tried to use the pre compiled version without luck. 

The settings in `config.yml` appear to be fine. I test the connection locally for MySQL and the user I've created with no issues. 

I've tried `java -jar target/commafeed.jar server config.yml` and `java -Djava.net.preferIPv4Stack=true -jar target/commafeed.jar server config.yml`.

```
$java -version
openjdk version "1.8.0_275"
OpenJDK Runtime Environment (build 1.8.0_275-8u275-b01-0ubuntu1~20.04-b01)
OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)
```
```
mysql --version  
mysql  Ver 8.0.22-0ubuntu0.20.04.3 for Linux on x86_64 ((Ubuntu))
```

Any guidance to solve this is appreciated. Thanks.

**Comment by Athou** [comment]
Seems like the connection to the database is not configured correctly. Do you mind sharing your config.yml file?

**Comment by Devl326789** [comment]
> Seems like the connection to the database is not configured correctly. Do you mind sharing your config.yml file?

So i've just had a chance to come back to it now. It appears to be running right now, we'll see how it goes. 

So I had two errors and I did not read up the error chain far enough. 
One was something about `sha2_password` I think it was, and the other `java.sql.SQLException: Unknown system variable 'query_cache_size'`

https://stackoverflow.com/questions/49984267/java-sql-sqlexception-unknown-system-variable-query-cache-size

I fixed it by bumping the version in `pom.xml` and rebuilding:
```
<dependency>
            <groupId>mysql</groupId>
            <artifactId>mysql-connector-java</artifactId>
            <version>8.0.13</version>
  </dependency>
```

It's still running now but the original reason why this was opened appears to be fixed. 

Thank you for the quick reply, I love commafeed, great job on it.
